test/extended/cli/explain.go

```diff
@@ -119,8 +119,8 @@ var (
 		{Group: "operator.openshift.io", Version: "v1", Resource: "kubeschedulers"},
 		{Group: "operator.openshift.io", Version: "v1", Resource: "openshiftcontrollermanagers"},
 		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecas"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogapiservers"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogcontrollermanagers"},
+		// {Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogapiservers"},
```

Comment:
is there a reason to comment them vs just removing them?

Reply:
@jwforres @shawn-hurley no, I saw others were commented out :) I'll remove them instead.